pkg/operator/pool/controller.go

```diff
@@ -17,46 +17,110 @@ Some of the code below came from https://github.com/coreos/etcd-operator
 which also has the apache 2.0 license.
 */
 
-// Package cluster to manage a rook cluster.
-package cluster
+// Package pool to manage a rook pool.
+package pool
```

Comment:
having each CRD type in its own package seems like overkill, but we can rearrange later